### DISCUSSION TITLE
reverse order in song select when sorting by date added

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#19099] [Ideas]
Coming from stable i expect newer beatmaps to be at the bottom when sorting by date added, however in lazer they are at the top. This should probably be reversed to match stable. Maybe an option to reverse the order in song select over all could also be cool.

**Comment by peppy** [comment]
That seems back to front to me? It was intentionally changed, and I think it's a better sort how we have it on lazer.

**Reply by respektive** [reply]
Hmm, it's probably just a matter of getting used to it. I just always find myself scrolling to the bottom to check recently added maps and not finding what I'm looking for :sweat_smile:

**Reply by WitherFlower** [reply]
I think the best solution would be to have filters behave like on osu-web (or the beatmaps listing screen), where the order is reversed if you click on the filter again : 
![image](https://user-images.githubusercontent.com/52672543/178691672-a1172b0a-575c-498e-81d2-24604ed905de.png)
![image](https://user-images.githubusercontent.com/52672543/178691731-ceefc2aa-1b5d-4955-926a-4ff54ecaf7e2.png)

**Reply by bdach** [reply]
@MrWither83 i'm pretty sure that the sort controls already work like you describe in the client.

**Reply by WitherFlower** [reply]
@bdach I was referring to the sort controls in Song Select, and proposing to add the "reversing behavior" to those
![image](https://user-images.githubusercontent.com/52672543/178805818-22a68a36-a58f-4c47-a010-6f0318a84e93.png)

**Comment by AkiraTenchi** [comment]
I would like to second the option part even just as a search bar filter as like reverse=true or something would be a game changer. Especially for the Last Played sort option, since i personally like to use that to go through maps i haven't played in a while, but this currently requires me to scroll down all the way again after every play.

**Comment by Fierioziy** [comment]
I'm also used to play 99% of the time using `Date Added` filter and scrolling to the bottom to find the most recent maps and scrolling to top for the older ones.

I know it is intentional that newest maps are at the top. So - I think that the best middleground would be adding a small icon for toggling sorting order (for *any* filter, not only `Date Added`).  
That would give more functionality and would satisfy users like us.  
Even a very small clickable arrow (up and down on click) would be enough and would take at most like 10 px of space next to sorting combobox
